migrations/users/full-stable.js

```diff
@@ -57,7 +57,7 @@ async function updateUser (user) {
 export default async function processUsers () {
   const query = {
     migration: { $ne: MIGRATION_NAME },
-    'auth.local.username': 'SabreTest',
+    'auth.local.lowerCaseUsername': 'SabreTest',
```

Comment:
was this intended? or some global replace happened 😬

Reply:
It was intentional, but I should have changed the username to match. We mainly use that field when we're testing migrations on our test accounts. 😂